pipeline/stages/test.yaml

```diff
@@ -32,22 +32,52 @@ stages:
           displayName: '${{parameters.env}} Virtual Machine Tests '
           steps:
 
+            - task: AzureCLI@2
```

Comment:
should this be a template instead either local or remote?

Reply:
moving to template ...